webapp-django/crashstats/base/jinja2/crashstats_base.html

```diff
@@ -143,6 +143,7 @@
 
     {% for status in status_messages %}
     <div class="status-message severity-{{ status.severity }}">
+        <span>(<time datetime="{{ status.date.isoformat() }}" class="ago">{{ status.date.strftime('%a, %b %d %H:%M %Z') }}</time>)</span>
```

Comment:
Because I made this mistake in Air Mozilla, please move this to a jinja templatetags function. 
So that, in this template, you can just write `{{ time_ago(status.date) }}` 
Sooner or later we might otherwise introduce this use in another template and then you need to write that complicated `strftime` thing again. Sooner or later you might have differences. If it's ALL going through 1 simple template tag function it's easier to keep things clean.